api/src/main/java/org/eclipse/microprofile/config/spi/ConfigSourceInterceptor.java

```diff
@@ -0,0 +1,5 @@
+package org.eclipse.microprofile.config.spi;
+
+public interface ConfigSourceInterceptor {
+    String getValue(ConfigSourceInterceptorContext context) throws Exception;
```

Comment:
The return of this should probably be a `ConfigValue` I would think, and it should also accept a configuration key as a parameter (see my comment on `ConfigSourceInterceptorContext.getKey()`).  I'd drop `throws Exception` (see my comment on `ConfigSourceInterceptorContext.proceed()`.

Reply:
Ops, I did create the class but forgot to use it in the return for this method.